specs/commandline/verify.md

```diff
@@ -2,7 +2,19 @@
 
 ## Description
 
-Use `notation verify` command to verify signatures on an artifact. Signature verification succeeds if verification succeeds for at least one of the signatures associated with the artifact. The digest of the supplied artifact is returned upon successful verification. It is recommended that this digest reference be used to pull the artifact subsequently, as registry tags may be mutable, and a tag reference can point to a different artifact that what was verified.
+Use `notation verify` command to verify signatures associated with the artifact. Signature verification succeeds if verification succeeds for at least one of the signatures associated with the artifact. Upon successful verification, the output message is printed out as follows:
+
+```text
+Successfully verified for <registry>/<repository>@<digest>
+```
+
+Tags are mutable and a tag reference can point to a different artifact than that was signed referred by the same tag. If a `tag` is used to identify the OCI artifact, the output message is as follows:
+
+```text
+Resolved artifact tag '<tag>' to digest '<digest>' before verification.
+Warning: The resolved digest may not point to the same signed artifact, since tags are mutable.
+Successfully verified for <registry>/<repository>@<digest>
+```
```

Comment:
Should we insert the word "signature" in the text to say "Successfully verified **_signature_** for <registry>/<repository>@<digest>. This way as and when other tools like Docker client or Ratify incorporate notation experience they can just use the same output the Notation CLI client uses?